### DISCUSSION TITLE
Does terragrunt `sops_decrypt_file` function support `age`?

### DISCUSSION THREAD

**Discussion by kholisrag** [gruntwork-io/knowledge-base#555] [Help] [answered]
Does terragrunt `sops_decrypt_file` function support `age`?
https://github.com/FiloSottile/age

---

<ins datetime="2022-09-09T15:07:13Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109230">Tracked in ticket #109230</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
`sops_decrypt_file` uses https://github.com/mozilla/sops as a library to decrypt SOPS encrypted files, so unless `sops` has native support for `age`, the function will not support it.

**Reply by kholisrag** [reply]
well `age` supported by `sops` thought.
I will assume `sops_decrypt_file` support `age` for now, and test it later.

**Reply by kholisrag** [reply]
double checked, and its supported!
Thank you
